tests/bug-7725/test.yaml

```diff
@@ -1,5 +1,5 @@
 requires:
-  min-version: 8
+  min-version: 7
```

Comment:
nit: I like the idea of adding exact minimum version here, so 7.0.11

Reply:
I was told force-push is ok for SV